packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -118,25 +101,26 @@ export default class QuanticSort extends LightningElement {
     );
   }
 
-  get largest() {
-    return CoveoHeadless.buildFieldSortCriterion(
-      'size',
-      CoveoHeadless.SortOrder.Descending
-    );
-  }
-
   get options() {
     return [
-      {label: this.labels.relevancy, value: 'relevancy'},
-      {label: this.labels.newest, value: 'newest'},
-      {label: this.labels.oldest, value: 'oldest'},
+      {
+        label: this.labels.relevancy,
+        value: CoveoHeadless.buildCriterionExpression(this.relevancy),
```

Comment:
I'd suggest to store the options array in a constant instead of rebuilding it every time we access the `options` property.